README.developers.md

```diff
@@ -217,22 +217,52 @@ There are 4 main regression tests:
     QoR checks in this regression test are primarily 'canary' checks to catch gross degradations in QoR.
     Occasionally, changes can cause QoR failures (e.g. due to CAD noise -- particularly on small benchmarks); usually such failures are not a concern if the QoR differences are small.
 
-* `vtr_reg_nightly`: ~6 hours with `-j3`
+* `vtr_reg_nightly_test1`: ~4.5 hours with `-j8`
 
     **Goal:** Basic QoR and Performance evaluation.
 
     **Feature Coverage:** Medium
 
     **Benchmarks:** Small-medium size, diverse. Includes:
 
+    * ISPD benchmarks
     * MCNC20 benchmarks
+    * VTR benchmarks
+
+    **Architectures:** A wider variety of architectures
+
+   QoR checks in this regression are aimed at evaluating quality and run-time of the VTR flow.
+   As a result any QoR failures are a concern and should be investigated and understood.
+   
+* `vtr_reg_nightly_test2`: ~6 hours with `-j8`
+
+    **Goal:** Basic QoR and Performance evaluation.
+
+    **Feature Coverage:** Medium
+
+    **Benchmarks:** Small-medium size, diverse. Includes:
+
     * VTR benchmarks
     * Titan 'other' benchmarks (smaller than Titan23)
```

Comment:
Doesn't it include most of the titan23 benchmarks too?

Reply:
I think this documentation could be refactored to be shorter.  State all the things that pertain to all vtr_nightly<num> tests together, and then state only the details that are different for each sub-test.  Essentially these comprise one set of tests that should be checked before a PR (and are checked by CI) but they are split into sub-suites for more parallel execution during CI.

Reply:
Sure. I made changes based on the suggestions